common/res/d415.h

```diff
@@ -1,14 +1,18 @@
 // License: Apache 2.0. See LICENSE file in root directory.
 // Copyright(c) 2018 Intel Corporation. All Rights Reserved.
 
-// This file is auto-generated from d415.obj
+// This file is auto-generated from d415.obj using rs-embed tool
+// Manual changes added to support upgrade of LZ4 3-rd party deprecated function)
```

Comment:
I think we should change rs-embed to output these manual changes.
I think we should automatically run rs-embed and generate these files if realsense2-gl is compiled.

Reply:
And I think these do not belong in common/res... but rather in realsense2-gl?